build.sbt

```diff
@@ -368,6 +370,20 @@ lazy val codegenPlugin = (projectMatrix in file("modules/codegen-plugin"))
     scriptedBufferLog := false
   )
 
+lazy val cli = (projectMatrix in file("modules/cli"))
+  .settings(
+    name := "smithy4s-cli",
+    testFrameworks += new TestFramework("weaver.framework.CatsEffect"),
```

Comment:
I think this can be enabled in all modules by default. It doesn't actually add any dependencies.

Reply:
I added it to the top of the build file